src/modules/Calendar/CalendarMonth.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+import '../../styles/CalendarStyles/CalendarMonth.css'
+
+import moment, {weekdaysShort} from "moment";
+import CalendarDay from "./CalendarDay";
+
+
+class calendarMonth extends React.Component {
+    generate(firstDay){
+        let month = [];
+
+        let firstWeek = Array(7).fill('').map((v,i)=>{
+            return i < firstDay.get("weekday") ? 0 : Math.floor(i-firstDay.get("weekday") + 1).toString();
+        });
+        month.push(firstWeek);
+
+        let nextDay = 7 - firstDay.get("weekday") + 1;
+        let numOfWeeks = Math.ceil((moment(firstDay).daysInMonth() - nextDay) / 7);
+
+        for (let w=0; w < numOfWeeks; w++){
+            month.push(Array(7).fill().map((v, i) => {
+                let day = (w * 7) + nextDay + i;
+                return day < moment(firstDay).daysInMonth() ? day.toString() : "0";
+            }))
+        }
+        return month;
+    }
+
+    selectDateInCalendar = (dataIndex) => {
+        this.props.updateSelectedDate(dataIndex, this.props.firstDay.get("month"));
+    };
+
+    renderWeek(arr) {
+        return (
+            <div className={"calendar__month__week"} key={arr[0]}>
+
+                { arr.map( (value, index) => <CalendarDay date={value}
```

Comment:
don't put extra spaces in your code, it's not always good to separate rows by empty lines :)